client/src/pages/TaskFormPage/TaskFormPage.js

```diff
@@ -85,54 +85,59 @@ const TaskFormPage = () => {
 
   return (
     <div className={styles.bg}>
-      <p className={styles.header}>Create a Task</p>
-      <div className={styles.inputBox}>
-        <InputBox
-          header={"Task Title"}
-          type={"text"}
-          placeholder={"Task Name"}
-          onChange={(e) => updateTaskFormData(e, "title")}
-          value={taskFormData.title}
-        />
-        <InputBox
-          header={"Location"}
-          type={"text"}
-          placeholder={"Location Name"}
-          onChange={(e) => updateTaskFormData(e, "location")}
-          value={taskFormData.location}
-        />
-        <InputBox
-          header={"Description"}
-          type={"text"}
-          placeholder={""}
-          expanded={true}
-          onChange={(e) => updateTaskFormData(e, "description")}
-          value={taskFormData.description}
-        />
-        <p className={styles.inputHeader}>Start</p>
-        <DateSelector
-          showTime={true}
-          selectedDate={taskFormData.startDate}
-          onDateChanged={(newDate) => updateTaskFormData(newDate, "startDate")}
-        />
-
-        <p className={styles.inputHeader}>End</p>
-        <DateSelector
-          showTime={true}
-          selectedDate={taskFormData.endDate}
-          onDateChanged={(newDate) => updateTaskFormData(newDate, "endDate")}
-        />
-      </div>
+      <div>
+        <p className={styles.header}>Create a Task</p>
+        <div className={styles.inputBox}>
+          <InputBox
+            header={"Task Title"}
+            type={"text"}
+            placeholder={"Task Name"}
+            onChange={(e) => updateTaskFormData(e, "title")}
+            value={taskFormData.title}
+          />
+          <InputBox
+            header={"Location"}
+            type={"text"}
+            placeholder={"Location Name"}
+            onChange={(e) => updateTaskFormData(e, "location")}
+            value={taskFormData.location}
+          />
+          <InputBox
+            header={"Description"}
+            type={"text"}
+            placeholder={""}
+            expanded={true}
+            onChange={(e) => updateTaskFormData(e, "description")}
+            multiline={true}
+            value={taskFormData.description}
+          />
+          <p className={styles.inputHeader}>Start</p>
+          <DateSelector
+            showTime={true}
+            selectedDate={taskFormData.startDate}
+            onDateChanged={(newDate) =>
+              updateTaskFormData(newDate, "startDate")
+            }
+          />
+
+          <p className={styles.inputHeader}>End</p>
+          <DateSelector
+            showTime={true}
+            selectedDate={taskFormData.endDate}
+            onDateChanged={(newDate) => updateTaskFormData(newDate, "endDate")}
+          />
+        </div>
 
-      {errorMessage ? <ErrorMessage errorMessage={errorMessage} /> : <></>}
+        {errorMessage ? <ErrorMessage errorMessage={errorMessage} /> : <></>}
```

Comment:
Minor comment: I would probably add some extra margin at the bottom (so the errors are spaced out a bit better)
![image](https://user-images.githubusercontent.com/24628243/163659520-c4050863-6b8b-44e6-bece-de5189bb9f5e.png)

Reply:
Added 10px margin.